doc/development/deprecations.rst

```diff
@@ -6,6 +6,12 @@ Deprecations
 Pending deprecations
 --------------------
 
+* The ``qml.utils.decompose_hamiltonian()`` method is removed. Please
+  use ``qml.pauli_decompose()``.
+
+  - Still accessible in v0.27
+  - Will be removed in v0.28
+
```

Comment:
I we removed the method, shouldn't we delete this from the deprecations page?

Reply:
I want this to be a place when people are wondering "why did that function go?" So we need an entry.

But maybe we can change the wording, since its directly removed instead of going through a deprecation cycle.

Reply:
Moved to the correct section 👍🏼